scripts/start-deployFabric

```diff
@@ -70,8 +70,12 @@ if [[ ! -e ${SERVER} ]]; then
     exit 10
   fi
 
+  mv server.jar "server-${VANILLA_VERSION}.jar"
   mv fabric-server-launch.jar "${SERVER}"
 fi
 
+# Specify which server jar to run
+echo "serverJar=server-${VANILLA_VERSION}.jar" >  fabric-server-launcher.properties
```

Comment:
Change this to a debug log or remove

Reply:
I'm sorry, for some reason (maybe because I was using the github mobile app), my eyes stopped before the redirect.